frontend/components/Dashboard/MyFeed/MyFeed.tsx

```diff
@@ -32,8 +32,9 @@ const MyFeed: React.FC<Props> = ({ currentUser }) => {
   const showPagination = count > NUM_POSTS_PER_PAGE
   const pageTitle = 'My Feed'
 
+  console.log(count)
```

Comment:
I should have the pagination bug fixed shortly so we can remove this console log and potentially the comment on line 37, if you want!

Reply:
Cool cool! Done!